orderBook.js

```diff
@@ -0,0 +1,17 @@
+const isCompatible = (existingOrder, incomingOrder) => existingOrder.type !== incomingOrder.type && existingOrder.price >= incomingOrder.price
+
```

Comment:
You only want the existingOrder price to be higher if it's the buy. If it is a sell then you want the incomingOrder price to be larger.